test/common/network/address_impl_test.cc

```diff
@@ -25,24 +25,30 @@ void makeFdBlocking(int fd) {
   ASSERT_EQ(::fcntl(fd, F_SETFL, flags & (~O_NONBLOCK)), 0);
 }
 
-void testSocketBindAndConnect(const Instance& loopbackPort) {
+void testSocketBindAndConnect(const std::string& addr_port_str) {
+  auto addr_port = parseInternetAddressAndPort(addr_port_str);
+  ASSERT_FALSE(addr_port == nullptr);
```

Comment:
Sorry, meant to go back and get the rest after confirming that the first one that I changed compiled.